models/articles-model.js

```diff
@@ -0,0 +1,20 @@
+const connection = require('../db/connection');
+const topicData = require('../db/data/test-data/articles');
+
+
+const selectArticle = (article_id) => {
+
+    if (isNaN(article_id)) {
+        return Promise.reject({status: 400})
+    };
```

Comment:
If you want here you can remove this logic to reject the Promise...
PSQL will throw this error automatically if the ID is not a number, and the `$1` protects against sql injection